### PR TITLE
integrate topoflow climate

### PR DESCRIPTION
**Pull request by binh-vu**

### CODE REVIEW

File: dtran/dcat/api.py [hunk 1]
```diff
@@ -321,3 +321,25 @@ def handle_api_response(response: requests.Response):
             """
 
             raise Exception(msg)
+
+
+if __name__ == '__main__':
+    # Update GLDAS representation
+    provenance_id = os.environ['PROVENANCE_ID']
+    dsmodel = "/home/rook/workspace/mint/MINT-Transformation/examples/topoflow4/dev/gldas.yml"
```
Comment (minhptx) [inline dtran/dcat/api.py:329]: Absolute path here

File: examples/topoflow4/topoflow_climate.yml [hunk 2]
```diff
@@ -4,10 +4,11 @@ adapters:
...
     adapter: funcs.Topoflow4ClimateWriteFunc
     inputs:
-      input_dir: "/tmp/demo/input/weather"
-      temp_dir: "/tmp/demo/tmp"
-      output_file: "/tmp/demo/output/climate_all.zip"
-      var_name: precipitation
-      DEM_bounds: "23.995416666666, 6.532916666667, 28.020416666666, 9.566250000000"
-      DEM_xres_arcsecs: "30"
-      DEM_yres_arcsecs: "30"
+      input_dir: "/home/rook/workspace/mint/MINT-Transformation/data/GLDAS"
```
Comment (minhptx) [inline examples/topoflow4/topoflow_climate.yml:7]: This one is currently used as example for MITRE test. All of the paths should be changed

**Comment by dgarijo** [comment]
A few days ago Scott mentioned that the RTI files for GLDAS transformation was not correct. Are the changes included here?

**Comment by binh-vu** [comment]
@dgarijo yes, it should be, as I just directly call his code.

**Pull request merged**